### ISSUE TITLE
[BUG] Unable to use ARCtrl.Json in fable JavaScript environment

### ISSUE THREAD

**Issue by Freymaurer** [nfdi4plants/ARCtrl#373]
We use conditional references for environment targetet json libs:

![image](https://github.com/nfdi4plants/ARCtrl/assets/39732517/84b38669-c4e3-45f0-bad5-0e502a7c458d)

This works for native transpilation and testing, but not when referenced in another fable environment.

**Comment by Freymaurer** [comment]
More:
![image](https://github.com/nfdi4plants/ARCtrl/assets/39732517/0362f6d4-1650-479b-bd76-9f1eccc92c8d)

**Comment by Freymaurer** [comment]
we hotfixed this with ARCtrl.alpha.6.swate hotfix. This version does not include conditional python/javascript json dependency, instead it always depens on thoth.json.javascript

**Comment by Freymaurer** [comment]
To publish swate prerelease, you can use:

1. change ARCtrl.Json.fsproj dependencies to
```xml
<ItemGroup>
    <PackageReference Include="NJsonSchema" Version="10.8.0" />
    <PackageReference Include="Thoth.Json.Core" Version="0.2.1" />
    <PackageReference Include="Thoth.Json.Newtonsoft" Version="0.1.0" />
    <PackageReference Include="Thoth.Json.JavaScript" Version="0.1.0" />
  </ItemGroup>
```
2. `build.cmd prerelease`
3. on request type `swate.b`
4. on request give prerelease version

**Comment by HLWeil** [comment]
I'm currently investigating why it doesn't work in Fable. '

The problem lies already with the dependency resolver they use, https://github.com/phmonte/Buildalyzer.

**Comment by HLWeil** [comment]
Should be fixed since `>=2.1.0-alpha.1` (#446)

@Freymaurer, can you confirm and close Issue?

**Comment by Freymaurer** [comment]
With ARCtrl.JavaAcript it is now working as intended
